src/libs/ReportUtils.ts

```diff
@@ -3065,10 +3065,41 @@ function getInvoicePayerName(report: OnyxEntry<Report>): string {
     return getPolicyName(report, false, allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${invoiceReceiver?.policyID}`]);
 }
 
+/**
+ * Parse html of reportAction into text
+ */
+function parseReportActionHtmlToText(reportAction: OnyxEntry<ReportAction>, reportID: string): string {
+    const {html, text} = reportAction?.message?.[0] ?? {};
+
+    if (!html) {
+        return text ?? '';
+    }
+
+    const mentionReportRegex = /<mention-report reportID="(\d+)" *\/>/gi;
+    const matches = html.matchAll(mentionReportRegex);
+
+    const reportIdToName: Record<string, string> = {};
+    for (const match of matches) {
+        if (match[1] !== reportID) {
+            // eslint-disable-next-line @typescript-eslint/no-use-before-define
+            reportIdToName[match[1]] = getReportName(getReport(match[1])) ?? '';
+        }
+    }
+
+    const mentionUserRegex = /<mention-user accountID="(\d+)" *\/>/gi;
+    const accountIdToName: Record<string, string> = {};
+    const accountIds = Array.from(html.matchAll(mentionUserRegex), (m) => Number(m[1]));
+    const logins = PersonalDetailsUtils.getLoginsByAccountIDs(accountIds);
+    accountIds.forEach((id, index) => (accountIdToName[id] = logins[index]));
+
+    const parser = new ExpensiMark();
+    return parser.htmlToText(html, {reportIdToName});
```

Comment:
I am using ExpensiMark.htmltoText instead of my original code.
But there is typecheck error for `accountIdToName` of extras in [here](https://github.com/Expensify/expensify-common/blob/84ac0394ab1021d3d892338c4dbb3f87c05b0bc9/lib/ExpensiMark.d.ts#L39). The field must be `accountIdToName` and not `accountIDToName`. 
The typecheck will fails if I include `accountIdToName` in this line.

Reply:
We need to include `accountIdToName`, don't we?

Reply:
@luacmartins  Yes, but there will be lint error if I used `accountIdToName`. 
the field is defined as `accountIDName` in [here](https://github.com/Expensify/expensify-common/blob/84ac0394ab1021d3d892338c4dbb3f87c05b0bc9/lib/ExpensiMark.d.ts#L39).

But in the the javascript function it uses `accountIdName` with lower `d` in [here](https://github.com/Expensify/expensify-common/blob/c3cc2c0d07c961df08a7147b9c4405014521b8bd/lib/ExpensiMark.js#L552C53-L552C68).

Should I make changes in `ExpensiMark` ?

Reply:
Yes, let's do it.